src/jupiterone/entities/ServiceEntity.ts

```diff
@@ -0,0 +1,12 @@
+import { EntityFromIntegration } from "@jupiterone/jupiter-managed-integration-sdk";
+
+export const SERVICE_ENTITY_CLASS = ["Service", "Control"];
+export const SERVICE_ENTITY_TYPE = "onelogin_service";
+
+export interface StandardizedOneloginServiceEntity
```

Comment:
Let's rename this to drop the `Standardized` prefix. I know the Okta integration uses that all over the place, but I think we're going to work to rename those as well. It does not communicate anything meaningful, I think, and none of the other types in this project start with that prefix.

I think we can also drop the `Onelogin` prefix, as no other entity types in this project start with that prefix. For example, there is an `AppEntity`, instead of `OneLoginAppEntity`.

This will make the file name `ServiceEntity` more correct as well 👍 

```suggestion
export interface ServiceEntity
```